cocos2d/videoplayer/video-player-impl.js

```diff
@@ -158,23 +158,24 @@ let VideoPlayerImpl = cc.Class({
         video.style.height = height + 'px';
     },
 
-    _createDom () {
+    _createDom (mute, volume) {
```

Comment:
Can we just merge these args into muted? And pass (mute || volume === 0) directly when calling this function.

Reply:
Good idea, fixed this!